### PR TITLE
Wallet connect improvements

### LINKED ISSUE #5469: Wallet connect DAPP sign transaction is only triggered on “Copy Signature”
### Expected behavior
Let the DAPP know the transaction has successfully been signed. This should happen when password has been entered successfully not when clicking copy signature.

### Actual behavior
https://user-images.githubusercontent.com/8784876/266607400-b6912062-6aa2-4c0e-bba7-53163032c19d.mov 

(https://github.com/LiskHQ/lisk-desktop/issues/5302)

### PR DESCRIPTION
**Pull request by oskarleonard**
### What was the problem?

This PR resolves #5469 #5468 #5470

### How was it solved?

Changed flow of wallet connect responses

#### Notify on success
1. Initiate transaction from wallet connect
2. enter correct password
3. Expected: Wallet connect should receive response

#### Notify on cancel
1. Initiate transaction from wallet connect
2. Click cancel
3. Expected: Wallet connect should receive response

##### Allow requestView modal to receive new wallet connect requests
1. Initiate transaction from wallet connect
2. enter correct password
3. Initiate another transaction from wallet connect
4. Expected: New request should be seen in desktop wallet

### CODE REVIEW

File: src/modules/wallet/components/RequestSignStatus/index.js [hunk 1]
```diff
@@ -29,12 +29,21 @@ const successData = (t) => ({
   ),
 });
 
+const getStringifiedTransactionJSON = (signedTransaction) => {
+  const moduleCommand = joinModuleAndCommand(signedTransaction);
+  const paramSchema = moduleCommandSchemas[moduleCommand];
+  const transactionJSON = toTransactionJSON(signedTransaction, paramSchema);
+  return JSON.stringify(transactionJSON);
+};
```
Comment (eniolam1000752) [inline src/modules/wallet/components/RequestSignStatus/index.js]: I think it would be best to get `moduleCommandSchemas` from `useCommandSchema` hook.

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]

File: src/modules/wallet/components/RequestSignStatus/index.js (hunk 1 above)
Comment (oskarleonard) [reply src/modules/wallet/components/RequestSignStatus/index.js]: Do you think it is safe? This above code is how we did it previously. I just moved it out of the component. 
image

I am worried about loading state if we introduce that hook, could affect this component  🤔

Comment (oskarleonard) [reply src/modules/wallet/components/RequestSignStatus/index.js]: Looks like props contain the schema. Maybe we can use that instead 🤔 
image

image

Comment (eniolam1000752) [reply src/modules/wallet/components/RequestSignStatus/index.js]: hmmm okay

**Review by ManuGowda — COMMENTED** [review]
Screenshot 2023-11-22 at 8 50 37 AM

Now that we changed from signature to full transaction we should updated the text?

**Review by ManuGowda — APPROVED** [review]

**Review by ikem-legend — APPROVED** [review]

**Review by eniolam1000752 — APPROVED** [review]

**Pull request merged**